sources/packages/backend/apps/api/src/services/application-offering-change-request/application-offering-change-request.service.ts

```diff
@@ -605,6 +606,21 @@ export class ApplicationOfferingChangeRequestService {
       await transactionalEntityManager
         .getRepository(ApplicationOfferingChangeRequest)
         .save(applicationOfferingChangeRequest);
+      // Send the application offering change request completed notification.
+      const systemUser = await this.systemUsersService.systemUser();
+      await this.notificationActionsService.saveApplicationOfferingChangeRequestCompletedByMinistry(
+        {
+          givenNames:
+            applicationOfferingChangeRequest.application.student.user.firstName,
```

Comment:
As a suggestion, a const can be created to keep the `applicationOfferingChangeRequest.application.student`.